ChocolateyForBusinessQuickStartGuide.md

```diff
@@ -93,26 +93,28 @@ As with configuration managers, this is out-of-scope for this document. Generall
 
 1. From an Administrator PowerShell Window, run the following command to install Chocolatey:
 
-    ```powershell
-    Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
-    ```
+```powershell
+Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
+```
+

```

Comment:
Without the spaces, the numbering starts over. Was this intentional?

Reply:
Definitely unintentional 🤦 . Reverting now.